universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -198,6 +227,52 @@ private ContainerTag renderBlockList(
     return container;
   }
 
+  private ContainerTag blockMoveButtons(
+      Request request,
+      long programId,
+      ImmutableList<BlockDefinition> blockDefinitions,
+      BlockDefinition blockDefinition) {
+    String moveUpFormAction =
+        routes.AdminProgramBlocksController.move(programId, blockDefinition.id()).url();
+    // Move up button is invisible for the first block
+    String moveUpStyles =
+        StyleUtils.joinStyles(
+            AdminStyles.MOVE_BLOCK_BUTTON,
+            (blockDefinition.id() == blockDefinitions.get(0).id()) ? Styles.INVISIBLE : "");
```

Comment:
question: Does `INVISIBLE` make this button non-functional as well? Just want to verify.

Reply:
yup! it is like `hidden` except it still takes up the space, which is nice for formatting.